automated/build.sh

```diff
@@ -146,7 +151,7 @@ docker run $ARGS_UPLOAD $IMAGE /bin/bash -c "exec curator check-target-available
 #
 # * Upload the docs to S3
 # * Upload the new snapshot .yaml file to the appropriate Github repo, also upload its constraints
-docker run $ARGS_UPLOAD $IMAGE /bin/bash -c "curator upload-docs --target $TARGET && curator upload-github --target $TARGET"
+docker run $ARGS_UPLOAD $IMAGE /bin/bash -c "curator upload-docs --target $TARGET ${DOCS_BUCKET:+--bucket $DOCS_BUCKET}" && curator upload-github --target $TARGET"
```

Comment:
Reverted the PR because in this line the " doesn't match - also I noticed quite strange log messages during running this script mentioning a LTS while I was running a nightly build. Hope this had no unwanted side effects.

Reply:
Oops, good catch. Of course I modified this line *after* running the script locally. You can't write a single line of bash without introducing a bug 😄

Reply:
Probably good idea to run it through shellcheck also.
(Though it doesn't catch the undefined NOPLAN.)